src/k8s/pkg/k8sd/features/calico/register.go

```diff
@@ -8,30 +8,30 @@ import (
 
 func init() {
 	images.Register(
-		fmt.Sprintf("%s/%s:%s", tigeraOperatorRegistry, tigeraOperatorImage, tigeraOperatorVersion),
+		fmt.Sprintf("%s/%s:%s", imageRepo, tigeraOperatorImage, tigeraOperatorVersion),
 	)
 
 	// TODO: configurable Calico images, include in this list
 	//
 	// Hardcoded list based on "k8s kubectl get node -o template='{{ range .items }}{{ .metadata.name }}{{":"}}{{ range .status.images }}{{ "\n- " }}{{ index .names 1 }}{{ end }}{{"\n"}}{{ end }}' | grep calico":
 	//
-	// - docker.io/calico/node:v3.28.0
-	// - docker.io/calico/cni:v3.28.0
-	// - docker.io/calico/apiserver:v3.28.0
-	// - docker.io/calico/kube-controllers:v3.28.0
-	// - docker.io/calico/typha:v3.28.0
-	// - docker.io/calico/node-driver-registrar:v3.28.0
-	// - docker.io/calico/csi:v3.28.0
-	// - docker.io/calico/pod2daemon-flexvol:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/node:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/cni:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/apiserver:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/kube-controllers:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/typha:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/node-driver-registrar:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/csi:v3.28.0
+	// - ghcr.io/canonical/k8s-snap/calico/pod2daemon-flexvol:v3.28.0
 
 	images.Register(
-		"docker.io/calico/node:v3.28.0",
-		"docker.io/calico/cni:v3.28.0",
-		"docker.io/calico/apiserver:v3.28.0",
-		"docker.io/calico/kube-controllers:v3.28.0",
-		"docker.io/calico/typha:v3.28.0",
-		"docker.io/calico/node-driver-registrar:v3.28.0",
-		"docker.io/calico/csi:v3.28.0",
-		"docker.io/calico/pod2daemon-flexvol:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/node:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/cni:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/apiserver:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/kube-controllers:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/typha:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/node-driver-registrar:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/csi:v3.28.0",
+		"ghcr.io/canonical/k8s-snap/calico/pod2daemon-flexvol:v3.28.0",
 	)
```

Comment:
this sort of deals with the TODO, so you can just put everything in a bit `images.Register()` call. also, make sure to use `imageRepo` everywhere instead of hardcoding it in some of the entries